<?xml version="1.0"?>
<launch>
    <arg name="PATH_TO_VOCABULARY" value="$(find slam)/vocab/ORBvoc.txt"/>
    <arg name="PATH_TO_SETTINGS_FILE" value="$(find slam)/config/Monocular-Inertial/RPVIO-Sim-bag.yaml"/>
    <arg name="PATH_TO_RESULT" value="/out/RPVIO/c1" />

    <node name="slam" pkg="slam" type="Dynamic_Mono"  
        args="$(arg PATH_TO_VOCABULARY) $(arg PATH_TO_SETTINGS_FILE) $(arg PATH_TO_RESULT)" 
        output="screen">
        <param name="init_delay" type="double" value="100"/>
        <param name="frame_delay" type="double" value="100"/>
        <param name="init_frame" type="int" value="2"/>     
        <param name="image_delay" type="int" value="2"/>     
        <param name="save_image_result" type="int" value="0"/> 
        <param name="use_viewer" type="int" value="1"/>      
        <param name="semantic_config_path" type="string" value="$(find slam)/config/yolact_settings.yaml"/>    
    </node>
</launch>
